pages/marketplace.rst

```diff
@@ -54,6 +54,17 @@ You cannot submit content that does not contain a ``LICENSE`` or ``COPYING`` fil
 We recommend to consult `ChooseALicense.com <http://choosealicense.com>`_ if you are
 unsure which license to use.
 
+4 Types of Add-Ons
+=====================
```

Comment:
Minor nitpick: The line length should match the text above (prints a warning when building the documentation with Sphinx).